simde/x86/avx512dq.h

```diff
@@ -204,6 +204,68 @@ simde_mm512_maskz_and_pd(simde__mmask8 k, simde__m512d a, simde__m512d b) {
 #define _mm512_maskz_andnot_pd(k, a, b) simde_mm512_maskz_andnot_pd(k, a, b)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m512
+simde_mm512_or_ps (simde__m512 a, simde__m512 b) {
+#if defined(SIMDE_X86_AVX512DQ_NATIVE)
+  return _mm512_or_ps(a, b);
+#else
+  simde__m512_private
+    r_,
+    a_ = simde__m512_to_private(a),
+    b_ = simde__m512_to_private(b);
+
+#if defined(SIMDE_X86_AVX_NATIVE)
```

Comment:
This should be

```c
#if SIMDE_NATURAL_VECTOR_SIZE_LE(256)
```

It's a relatively new feature so the function you copied from probably didn't use it, but you should use that for new code.